lib/css/ilr-admin.css

```diff
@@ -235,17 +235,10 @@ body.tools_page_invalid-login-redirect div.logging.add-on .badge {
 			line-height: 1.5;
 			color: #fff;
 			background: #333;
-			-webkit-box-shadow: 2px 2px 5px #aaa;
-			   -moz-box-shadow: 2px 2px 5px #aaa;
-				      box-shadow: 2px 2px 5px #aaa;
+			box-shadow: 2px 2px 5px #aaa;
 			opacity: 0;
-			-webkit-transition: opacity 250ms ease-out;
```

Comment:
@brothman01 Why was all of this removed? Can you add it back in and re-run grunt?